app/models/project.rb

```diff
@@ -1,4 +1,15 @@
 class Project < ApplicationRecord
   validates_presence_of :name, :material
+
   belongs_to :challenge
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects 
+
+  def contestant_count
+    self.contestants.count
+  end
+
+  def experience_average
+    contestants.average(:years_of_experience)
```

Comment:
Nice AR here! 🙌

Reply:
The only thing missing from these models are _model tests_. Whenever you write a new model method, you should either stop right away before writing the method and write a test for it (remember: TDD is red, green, refactor), or make sure the method you write has a test before shipping the code (in this case turning it in, or deploying it to the internet). Either way, your model test coverage should be 100% at all times.